### PR TITLE
Added a RN for additional information Insights Operator collects in 4.7

### PR DESCRIPTION
**Pull request by mmuehlfeldRH**
This is a combined RN for the following enhancements:
https://bugzilla.redhat.com/show_bug.cgi?id=1913356
https://bugzilla.redhat.com/show_bug.cgi?id=1914975
https://bugzilla.redhat.com/show_bug.cgi?id=1893953
https://bugzilla.redhat.com/show_bug.cgi?id=1907378
https://bugzilla.redhat.com/show_bug.cgi?id=1883425
https://bugzilla.redhat.com/show_bug.cgi?id=1885930
https://bugzilla.redhat.com/show_bug.cgi?id=1887778
https://bugzilla.redhat.com/show_bug.cgi?id=1886751
https://bugzilla.redhat.com/show_bug.cgi?id=1900496

@openshift/team-documentation , @codyhoag  please review and merge into [enterprise-4](https://issues.redhat.com/browse/enterprise-4).7. Thanks.

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -236,6 +236,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
 === Developer experience
 
 
+
+[id="ocp-4-7-insights-operator"]
+=== Insights Operator
+
+In {product-title} 4.7, the Insights Operator collects the following additional information:
```
Comment (codyhoag) [inline release_notes/ocp-4-7-release-notes.adoc:243]: For 4.6, we had this portion structured like this:
```
[id="ocp-4-6-insights-operator"]
=== Insights Operator

[id="ocp-4-6-insights-operator-data-collection-enhancements"]
==== Insights Operator data collection enhancements

In {product-title} 4.6, the Insights Operator collects the following additional information:
```
What are your thoughts on mimicking that here?

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -236,6 +236,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
...
+
+* The top 100 `InstallPlan` entries to identify invalid Operator Lifecycle Manager (OLM) installations
+* The service accounts from the Kubernetes default namespace and the `openshift*` built-in namespaces
+* The `ContainerRuntimeConfig` and `MachineConfigPools` configuration files to verify container storage limits
+* The configuration files for all available `operator.openshift.io` control pane resources to identify operators in unmanaged states
```
Comment (codyhoag) [inline release_notes/ocp-4-7-release-notes.adoc:248]: s/operators/Operators

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -236,6 +236,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
...
+* The `NetNamespaces` names, including their `netID` and egress IP addresses
+* The `StatefulSet` configuration definitions from the OpenShift default namespaces to check the Prometheus monitoring storage type
```
Comment (codyhoag) [inline release_notes/ocp-4-7-release-notes.adoc:250]: Does it make sense to change OpenShift &rarr; {product-title}?

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -236,6 +236,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
...
+* Appearances of certain log entries of pods in the `openshift-apiserver-operator` namespace
+* Appearances of certain log entries of `sdn` pods in the `openshift-sdn` namespace
+
```
Comment (codyhoag) [inline release_notes/ocp-4-7-release-notes.adoc:253]: I would lean towards ending each bullet point with a period.

**Review by codyhoag — COMMENTED** [review]
Just a few suggestions. Looks good otherwise!

**Comment by tremes** [comment]
@mmuehlfeldRH I would wait with merging this. The 4.7 GA was postponed and we will probably add some more interesting stuff.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (mmuehlfeldRH) [reply release_notes/ocp-4-7-release-notes.adoc:248]: Fixed.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (mmuehlfeldRH) [reply release_notes/ocp-4-7-release-notes.adoc:243]: ACK. I added the extra id and title.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 3 above)
Comment (mmuehlfeldRH) [reply release_notes/ocp-4-7-release-notes.adoc:250]: Updated.

File: release_notes/ocp-4-7-release-notes.adoc (hunk 4 above)
Comment (mmuehlfeldRH) [reply release_notes/ocp-4-7-release-notes.adoc:253]: They don't end with a period because they aren't full sentences.

**Comment by mmuehlfeldRH** [comment]
@codyhoag should we keep this request open and merge it before 4.7 GA? Or should we merge it and submit another PR if more data collection enhancements come in before GA?
I'm just asking because responsibilities have recently changed. @xJustin is the new writer for Insights Operator docs.
However, it's no problem for me to have an eye on this and update the PR before GA, if needed.

**Comment by codyhoag** [comment]
@mmuehlfeldRH I would prefer we merge it now so it's available for our previews and upcoming stakeholder review requests. It's a living document, so we expect many more updates overall before GA. Does that sound good?

**Comment by mmuehlfeldRH** [comment]
@codyhoag sure. Please merge if everything is Ok now.

**Comment by codyhoag** [comment]
LGTM! Merging...

**Pull request merged**